.spectral.yaml

```diff
@@ -26,6 +26,20 @@ rules:
           -  multipart/form-data
           -  application/octet-stream
 
+  must-return-content-types:
```

Comment:
Should me mention "accepted"-content-types because it is not just about returning a content type but being in the accepted list.

Reply:
That's why we provide description. I don't think there will be a case where the rule "name" only is referenced anywhere.